### PR TITLE
add documentation for use of conda environments

### LINKED ISSUE #704: Document use of environments with conda to reduce likelihood of user errors
Conda has a similar approach to virtual environments. We should specify that users of Conda should leverage this capability in our documentation and installation instructions. 

[https://conda.io/docs/user-guide/tasks/manage-environments.html](https://conda.io/docs/user-guide/tasks/manage-environments.html)

### PR DESCRIPTION
**Pull request by kne42**
add a section under the installation docs on how to set up virtual environments with conda

closes #704

### CODE REVIEW

**Comment by ambrosejcarr** [comment]
I haven't used conda in like 10 years. I'll try this tonight and let you know how I fare. :-)

File: docs/source/getting_started/installation/index.rst [hunk 1]
```diff
@@ -13,11 +13,11 @@ The output should look similar to this:
    % python --version
    Python 3.6.5
 
-Installation for users
-----------------------
-
+Using virtual environments
+--------------------------
+   
 Starfish names its dependencies and lists explicit versions, due to sensitivity to subtle algorithm
```
Comment (ttung) [inline docs/source/getting_started/installation/index.rst]: > lists explicit versions

I don't think this is still true today.  There is an option to install the versions we use for CI, but it's no longer mandatory.

Comment (kne42) [reply docs/source/getting_started/installation/index.rst]: ah, that's a good point! do we maybe want to just convert this into a strict/nonstrict installation guide? cc @ambrosejcarr @neuromusic

Comment (ambrosejcarr) [reply docs/source/getting_started/installation/index.rst]: Yes, I would add both options to this.

**Comment by ttung** [comment]
Landing!

**Pull request merged**